### ISSUE TITLE
Parse dependency manifest

### ISSUE THREAD

**Issue by kiliankoe** [kiliankoe/apodidae#16]
Two big advantages for actually reading and understanding the package manifest of a possible package:
 - Get a list of dependencies this package has.
 - ~Get a list of targets to try and tell if it's actually possible to include this within a project or if it's an executable.~ Seems out of scope.

(obviously depends on #13)
